vmad/lib/fastpm.py

```diff
@@ -526,7 +525,7 @@ def jvp(node, x1_, x2_, x1, x2):
         return dict(y_=x1.cdot(x2_).real + x2.cdot(x1_).real)
 
 class FastPMSimulation:
-    def __init__(self, stages, cosmology, pm, B=1, q=None):
+    def __init__(self, stages, Om0, cosmology, pm, B=1, q=None):
```

Comment:
Is Om0 here a symbol or a hyper-parameter?

If it is a hyper-parameter, why not use the Om0 parameter in cosmology?

Reply:
If it is a symbol, then there will be an error when the symbol is passed into MatterDominated pt.

Reply:
It is a hyper parameter, but is a symbol in the run autooperator. I just changed it so that the cosmology Om0 is called in the init. Thanks!